go.mod

```diff
@@ -1,8 +1,10 @@
 module github.com/beatlabs/patron
 
+go 1.16
```

Comment:
Maybe we should deal with the upgrade in another ticket. If we support 1.16 and after we are introducing again a breaking change and narrow the usage only to 1.16 out there . Can we leave it to what it was?

Reply:
I don't think it can be set to a lower version because Sarama requires it :thinking: so if you try to compile this project with anything below the Sarama v1.30 in vendor will not compile.

Only thing I can do is split this into two: one upgrades to Go v1.16 and one updates Sarama; let me know, it's not a lot of work for me to do that